src/main/java/net/dries007/tfc/client/GrassColorHandler.java

```diff
@@ -0,0 +1,143 @@
+package net.dries007.tfc.client;
+
+import java.awt.Color;
+import java.time.Month;
+import java.util.Random;
+
+import net.dries007.tfc.ConfigTFC;
+import net.dries007.tfc.util.calendar.CalendarTFC;
+import net.dries007.tfc.util.climate.ClimateTFC;
+import net.minecraft.block.state.IBlockState;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.math.MathHelper;
+import net.minecraft.world.ColorizerGrass;
+import net.minecraft.world.IBlockAccess;
+import net.minecraft.world.gen.NoiseGeneratorPerlin;
+
+
+public class GrassColorHandler 
+{
+	public static NoiseGeneratorPerlin noiseGenerator = new NoiseGeneratorPerlin(new Random("NOISE_GRASS".hashCode()), 2);
+	public static Color[] monthlyColors = new Color[12];
+
+    static
+    {
+    	resetColors();
+    }
+    
+    public static void resetColors()
+    {
+    	int julyCode = 0x00;
+    	int octoberCode = 0x00;
+    	int januaryCode = 0x00;
+    	int aprilCode = 0x00;
+    	try
+    	{
+        	julyCode = Integer.parseUnsignedInt(ConfigTFC.Client.GRASS_COLOR.seasonColorSummer, 16);
+        	octoberCode = Integer.parseUnsignedInt(ConfigTFC.Client.GRASS_COLOR.seasonColorAutumn, 16);
+        	januaryCode = Integer.parseUnsignedInt(ConfigTFC.Client.GRASS_COLOR.seasonColorWinter, 16);
+        	aprilCode = Integer.parseUnsignedInt(ConfigTFC.Client.GRASS_COLOR.seasonColorSpring, 16);
+    	}
+    	finally
+    	{
+    		monthlyColors[Month.JULY.ordinal()] = new Color(julyCode, true);
+        	monthlyColors[Month.OCTOBER.ordinal()] = new Color(octoberCode, true);
+        	monthlyColors[Month.JANUARY.ordinal()] = new Color(januaryCode, true);
+        	monthlyColors[Month.APRIL.ordinal()] = new Color(aprilCode, true);
```

Comment:
This will be overwritten by the for loop

Reply:
It won't. The loop doesn't touch months 0, 3, 6, 9 (January, March, July and October). It just uses them to blend the months in between. For example for i = 0, only month 0 + 1 (February) and 0 + 2 (March) will be set, January isn't changed.